### ISSUE TITLE
Install error when using install command from "Quickstart"

### ISSUE THREAD

**Issue by f-cozzocrea** [hpyproject/hpy#434]
While following the steps to install hpy in the "Quickstart" section of the docs, I get the following warning and errors:

```
WARNING: Generating metadata for package hpy.universal produced metadata for project name hpy. Fix your #egg=hpy.universal fragments.
Discarding git+https://github.com/hpyproject/hpy.git#egg=hpy.universal: Requested hpy from git+https://github.com/hpyproject/hpy.git#egg=hpy.universal has inconsistent name: filename has 'hpy-universal', but metadata has 'hpy'
ERROR: Could not find a version that satisfies the requirement hpy-universal (unavailable) (from versions: 0.0.0rc1)
ERROR: No matching distribution found for hpy-universal (unavailable)
```
Platform: Ubuntu 22.04
CPython version: 3.10
hpy version: 0.9.0rc2

Steps to reproduce:
`$ python3 -m pip install git+https://github.com/hpyproject/hpy.git#egg=hpy.universal
`

**Comment by f-cozzocrea** [comment]
@fangerer Sounds good, thanks for the tip! I've installed the pre-release and it's working for me.

**Comment by f-cozzocrea** [comment]
It looks like this issue was fixed in commit 310fde8. The development build installs without issue now.
